test/Interop/Cxx/reference/const-ref-parameter.swift

```diff
@@ -36,42 +36,34 @@ func testFunction() {
 
 // RUN: %target-swift-frontend -c -enable-experimental-cxx-interop -enable-objc-interop -I %S/Inputs %s -emit-silgen -o - | %FileCheck %s
 
-// COM: FIXME: should it be @in_guaranteed OptionsStruct? https://github.com/apple/swift/issues/60601
 // CHECK: [[FN1:%[0-9]+]] = function_ref @$sSo19OptionsConsumerObjCC7optionsABSo0A6StructV_tcfC : $@convention(method) (OptionsStruct, @thick OptionsConsumerObjC.Type) -> @owned OptionsConsumerObjC
```

Comment:
This one is the one that I was confused. IIUC `$sSo19OptionsConsumerObjCC7optionsABSo0A6StructV_tcfC` demangles to `__C.OptionsConsumerObjC.__allocating_init(options: __C.OptionsStruct) -> __C.OptionsConsumerObjC` which seems to be an Obj-C initializer, and it does not have `@in` nor `@in_guaranteed`. I don't know if this is required, or in this case it is  fine to pass a copy.

But this one is different than in your new test, which shows a non-mangled name. I wonder if the designated initializer bit changes something.